static/js/reducers/videoDetailUi_test.js

```diff
@@ -20,33 +16,14 @@ describe('VideoDetailUi', () => {
 
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
-    assertReducerResultState = createAssertReducerResultState(store, state => state.videoDetailUi);
     store = configureTestStore(rootReducer);
+    assertReducerResultState = createAssertReducerResultState(store, state => state.videoDetailUi);
   });
 
   afterEach(() => {
     sandbox.restore();
   });
 
-  it('should clear the edit dialog ui', () => {
```

Comment:
Maybe move some/all of these to `commonUi_test.js`:
```python
  it('sets the title', () => {
    assertReducerResultState(setEditVideoTitle, ui => ui.editVideoForm.title, '');
  });

  it('sets the description', () => {
    assertReducerResultState(setEditVideoDesc, ui => ui.editVideoForm.description, '');
  });
```

Reply:
Will do